src/routes/room/[id]/+page.svelte

```diff
@@ -72,6 +74,23 @@
 		});
 	});
 
+	async function generateVoice(message: Message) {
+		if (!message.translatedContent) {
+			return;
+		}
```

Comment:
We should actually use `message.translatedContent ?? message.content` probably, right?